src/Discord.Net.Rest/Entities/Channels/ChannelHelper.cs

```diff
@@ -76,15 +76,15 @@ internal static class ChannelHelper
         public static async Task<RestInviteMetadata> CreateInviteAsync(IGuildChannel channel, BaseDiscordClient client,
             int? maxAge, int? maxUses, bool isTemporary, bool isUnique, RequestOptions options)
         {
-            var args = new CreateChannelInviteParams { IsTemporary = isTemporary, IsUnique = isUnique };
-            if (maxAge.HasValue)
-                args.MaxAge = maxAge.Value;
-            else
-                args.MaxAge = 0;
-            if (maxUses.HasValue)
-                args.MaxUses = maxUses.Value;
-            else
-                args.MaxUses = 0;
+            if (maxAge.HasValue && maxAge > 86400)
+                throw new InvalidOperationException("The maximum age of an invite must be less than or equal to a day (86400).");
```

Comment:
IMO this would be better as an ArgumentOutOfRangeException

Reply:
Good idea, changed.